db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -1,85 +1,945 @@
+                       options: [
+                         { title: 'Ja' },
+                         { title: 'Nee' },
+                         { title: 'Weet ik niet' }
+                       ],
+                       show_otherwise: false,
+                       tooltip: 'Een deel van het regionaal budget voor VSV en kwetsbare jongeren komt via de contactgemeente naar de regio in de vorm van een specifieke uitkering met een specifiek doel (geoormerkt). Bij een decentralisatie-uitkering komt het bij de contactgemeente binnen \'op de grote hoop\', via het Gemeentefonds.'
+                     }, {
+                       id: :v77,
+                       hidden: true,
+                       type: :range,
+                       title: 'In hoeverre is een decentralisatie-uitkering wenselijk?',
+                       labels: ['Helemaal niet wenselijk', 'Heel wenselijk'],
+                     }, {
+                       id: :v78,
+                       hidden: true,
+                       type: :textarea,
+                       title: 'Wat voor gevolgen zou dat hebben voor uw werkzaamheden als RMC-coördinator?',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       id: :v79,
+                       hidden: true,
+                       type: :textarea,
+                       title: 'Wat voor gevolgen zou dat volgens u hebben voor de samenwerking met uw partners in de regio?',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       section_start: '<div class="divider"></div><p class="flow-text"><strong>Evaluatie register vrijstellingen leerplichtwet</strong></p>',
+                       id: :v80,
+                       hidden: true,
+                       type: :range,
+                       title: 'In hoeverre heeft het registreren van vrijstellingen eraan bijgedragen dat de betreffende jongeren niet meer onterecht als VSV-er werden benaderd?',
+                       labels: ['Geen bijdrage', 'Heel veel bijdrage']
+                     }, {
+                       id: :v81,
+                       hidden: true,
+                       type: :radio,
+                       title: 'Komt u nog steeds jongeren tegen die een vrijstelling hebben, maar toch als VSV-er worden geteld?',
+                       options: [
+                         { title: 'Ja', shows_questions: %i[v82] },
+                         { title: 'Nee' }
+                       ],
+                       show_otherwise: false
+                     }, {
+                       id: :v82,
+                       hidden: true,
+                       type: :textarea,
+                       title: 'Kunt u dit toelichten?',
+                       placeholder: 'Vul hier uw antwoord in'
                      }]
```

Comment:
misschien willen we dit beslist niet hoor, maar zou het nog interessant kunnen zijn om de deelnemers te vragen of ze nog algemene opmerkingen / feedback hebben? Een vrij tekst veld helemaal aan het eind?

Reply:
Over hun ervaring van het invullen van de vragenlijst bedoel je? Ja wellicht. Ik heb je comment in de google doc gezet. Als ze het daar mee eens zijn kunnen we het hier aan toevoegen.